### PR TITLE
Add console.js@0.0.2

### PR DESCRIPTION
**Pull request by Piicksarn**
@cdnjs/intern @cdnjs/intern2 , please help me checkout the pr, thanks!
I've already made a check for the files to including. :point_down: 
https://github.com/icodeforlove/Console.js/issues/9

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/icodeforlove/Console.js
- Official website (optional, not the repository): http://icodeforlove.github.io/Console.js/
- NPM package url (optional):  -
- GitHub / Bitbucket popularity (required):
  - Count of stars: 72
  - Count of watchers: 6
  - Count of forks: 8
- NPM download stats (optional):
  - Downloads in the last day: -
  - Downloads in the last week: -
  - Downloads in the last month: -
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [ ] The lib has notable popularity
  - [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by pvnr0082t** [comment]
@Piicksarn Could you try to ask author if it's possible to add git tag for the repo? Thanks.

**Comment by Piicksarn** [comment]
@cdnjs/intern @cdnjs/intern2 
I've it revased and the link shown below is for asking the git tags.
https://github.com/icodeforlove/Console.js/issues/10

**Comment by maruilian11** [comment]
1. please use `https://...` in repo url, which is better. 
2. you miss `homepage`.
3. it's easy to add v0.0.0 and v0.0.1 by tracking git commit log.
please also add them manually.

**Comment by PeterDaveHello** [comment]
> please use https://... in repo url, which is better.

When do we have this convention? Or, what's the reference saying https is better here? Looks no matter for me here.

**Comment by maruilian11** [comment]
yes, sorry, I made a mistake about the url type.

**Comment by Piicksarn** [comment]
@cdnjs/intern, @cdnjs/intern2
The lib had already been used with git tags, so I add the git autoupdate config for the pr.
Furthermore, i add the homepage config.

Thank you!

File: ajax/libs/console.js/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "console.js",
+  "filename": "console.min.js",
+  "version": "0.0.2",
+  "description": "A sane way to interface with your browser console and use the latest features.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/icodeforlove/console.js.git"
+  },
+  "homepage": "http://icodeforlove.github.io/Console.js/",
```
Comment (pvnr0082t) [inline ajax/libs/console.js/package.json]: @Piicksarn It will be better to use `https` instead of `http` for `github.io` domain.

**Comment by Piicksarn** [comment]
@pvnr0082t, Thanks!
I have modified the homepage config.
Please help me review it again, thank you!

**Comment by PeterDaveHello** [comment]
@Piicksarn you should mention the issue number in the PR comment!

**Comment by PeterDaveHello** [comment]
@Piicksarn and why didn't you rebase this branch with the latest master?

**Pull request merged**